build-support/python/clean.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
any reason this is not consistent with others, i.e. `#!/usr/bin/env bash`?

Reply:
nothing here uses BASH specific things so I just chose /bin/sh. I could make it use bash for consistency though